spec/cassettes/OpenTok_Streams/get_specific_stream_information.yml

```diff
@@ -0,0 +1,42 @@
+---
+http_interactions:
+- request:
+    method: get
+    uri: https://api.opentok.com/v2/project/123456/session/SESSIONID/stream/STREAMID
+    body:
+      encoding: US-ASCII
+      string: ''
+    headers:
+      User-Agent:
+      - OpenTok-Ruby-SDK/3.0.4.alpha1-Ruby-Version-2.5.1-p57
```

Comment:
The Ruby version should be dynamic if not it cause an error with a different version: - OpenTok-Ruby-SDK/<%= version %>